build.sh

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-REV=340070
+REV=343360
```

Comment:
Would it be helpful to add a comment here for where this revision number should come from?

Reply:
Thanks.